examples/autokey/autokey-setup/README.md

```diff
@@ -0,0 +1,25 @@
+# Autokey Example
+
```

Comment:
[Importing Autokey Key Handles Guidance](https://github.com/terraform-google-modules/terraform-google-kms/blob/master/docs/importing_autokey_key_handles.md) will be impacted with this PR. The migrating instructions provided there won't work anymore.

Since the changes in this PR are totally removing the Key Handle creation responsibility from `terraform-google-kms` module, I would recommend deleting that guidance file, since it won't make sense to provide an importing path to a resource that the KMS terraform module won't be creating/managing anymore. (If we delete that, we should also delete the [scripts folder](https://github.com/terraform-google-modules/terraform-google-kms/tree/master/scripts))

Reply:
make sense, keeping this comment open till I have approval from all of you reviewers.

Reply:
decided to not support the migration as the older module is not aligned with our current stand.